configure

```diff
@@ -0,0 +1,361 @@
+#!/bin/sh
+### Custom configure script of Dinit.
+## Initial prepartion
+set -eu
+cd "$(dirname "$0")"
+
+## Helper functions
+# According to POSIX, echo has unspecified behavior if any of its arguments contain “\” or
```

Comment:
This text is copied verbatim from the linked page - please don't do that, it raises copyright issues (unless you have specific permission to use the text). Just summarise the issue in your own words; a short sentence or two is fine.